ArduPlane/navigation.cpp

```diff
@@ -397,9 +397,10 @@ void Plane::update_fbwb_speed_height(void)
         int32_t alt_change_cm = g.flybywire_climb_rate * elevator_input * dt * 100;
         change_target_altitude(alt_change_cm);
```

Comment:
Your loosing this none zero alt change due to the reset. Previously it would always be zero because the reset would only trigger for `elevator_input` of zero. I think just putting the reset above the target change would fix.

Reply:
That's true. I don't think it makes much difference, because
  - the reset will typically be a bigger target altitude jump than this increment, so one could argue that the increment is done anyway
  - integrating your elevator input will not correspond to any target altitude anymore anyway, the reset makes the integral (altitude target) basically random
  - with the elevator jittering around the deadzone it would probably be better to keep the altitude target steady and have it not jitter up and down as well
  - the actual increment we loose this way is typically in the order of centimeters

That being said, I understand you prefer this increment being included. I also do think that resetting before applying change will safely address this. I just pushed an update.

Reply:
> * with the elevator jittering around the deadzone it would probably be better to keep the altitude target steady and have it not jitter up and down as well

`get_control_in` should ignore changes within the deadzone and return 0.

I agree its not a big change, but including it means we will be very slightly faster to respond. It is also the current behaviour.